test/lib/connectors/repos/change-reasons.js

```diff
@@ -43,7 +43,7 @@ experiment('lib/connectors/repos/change-reasons', () => {
     });
 
     test('collection is sorted on position property', async () => {
```

Comment:
We may need to tweak the test description to 
`sorted on description` 
to match the test

Reply:
Oh yes! Thank you :)